### ISSUE TITLE
Fields are missing

### ISSUE THREAD

**Issue by bhousel** [openstreetmap/iD#3356]
<img width="374" alt="screenshot 2016-08-16 14 14 27" src="https://cloud.githubusercontent.com/assets/38784/17710564/c914b2fe-63bb-11e6-9ead-6da4904535c4.png">

This is caused by an error in `preset.js` line 87:
`fieldsArr = [UIField(context.presets().field('name'), entity)];`

This line throws the error:
`TypeError: fields$1[field.type] is not a function`

and indeed the fields object contains

![screenshot 2016-08-16 14 13 37](https://cloud.githubusercontent.com/assets/38784/17710627/0e9c189e-63bc-11e6-9053-9fc640893b40.png)

So somehow `name` contains a string "[object Object]" instead of the actual object.

**Comment by Psigio** [comment]
Hi - I've seen this happen on my local build too.  I think it is possible an issue in rollup-plugin-json with regards to using "name" as it appears to be a globally-scoped variable already present in Chrome.  I've raised an issue at https://github.com/rollup/rollup-plugin-json/issues/13

With a bit of local testing, I found that if I added "name" to the list of reserved words used by rollup-plugin-json the issue seems to resolve, so I think it might just be down to an identifier clash.
